.ci.yaml

```diff
@@ -375,6 +383,15 @@ targets:
         [
           "ios-16-0_14a5294e"
         ]
+      $flutter/osx_sdk : >-
+        {
+          "sdk_version": "14e222b",
+          "runtime_versions":
+            [
+              "ios-16-4_14e222b",
+              "ios-16-2_14c18"
```

Comment:
These aren't right for this branch, I think you want 14a5294e .. not sure about the runtime_versions.